trustpinning/certs_test.go

```diff
@@ -363,6 +363,26 @@ func TestValidateRootWithPinnedCertAndIntermediates(t *testing.T) {
 		}
 	}
 	require.Equal(t, typedSignedRoot, validatedRoot)
+
+	// test is also works with a wildcarded gun in certs
+	validatedRoot, err = trustpinning.ValidateRoot(
+		nil,
+		signedRoot,
+		"docker.io/notary/test",
+		trustpinning.TrustPinConfig{
+			Certs: map[string][]string{
+				"docker.io/notar*": {ecdsax509Key.ID()},
+			},
+			DisableTOFU: true,
+		},
+	)
```

Comment:
Similar to this, can we also add a test that the more specific glob takes precedence over the less specific glob?
e.g. 

```
			Certs: map[string][]string{
 				"docker.io/notary/te*": {"badID"},
 				"docker.io/notar*":      {ecdsax509Key.ID()},
 			},
```

fails while 

```
			Certs: map[string][]string{
 				"docker.io/notary/te*": {ecdsax509Key.ID()},
 				"docker.io/notar*":      {"badID"},
 			},
```

succeeds?